getting_started/dedicated_administrators.adoc

```diff
@@ -0,0 +1,110 @@
+permissions and access to all user-created projects in your organization's
+cluster. While logged in to an account with this role, the basic
+link:../cli_reference/basic_cli_operations.html[developer CLI] (the `oc`
+command) allows you increased visibility and management capabilities over
+objects across projects, while the
+link:../cli_reference/admin_cli_operations.html[administrator CLI] (commands
+under the `oc adm` command, and formerly the `oadm` command) open up additional
+operations.
+
+[[gs-dedicated-admin-downloading-the-cli]]
+== Downloading the CLI
+
+You can download the CLI used for both normal developer operations and
+administrator operations from the *About* page in the web console. See
+link:../cli_reference/get_started_cli.html[Get Started with the CLI] for more
+detailed installation steps.
+
+[[gs-dedicated-admin-logging-in]]
+== Logging In and Verifying Permissions
+
+To log in as an administrator of a {product-title} cluster, you can log in to
+the web console or CLI just as you would if you were a normal developer user.
+
+When you link:../dev_guide/authentication.html#web-console-authentication[log
+into the web console], you will see all user-created projects across the
+cluster.
+
+You can log in using the CLI just like any other user, as well:
+
+----
+$ oc login <your_instance_url>
+----
+
+You can run the following command against an existing project to view the
+default link:../architecture/additional_concepts/authorization.html[policy
+bindings]. If you are a cluster administrator, you will see your account listed
+under *RoleBinding[dedicated-project-admin]*:
+
+====
+----
+$ oc describe policyBindings :default -n <any_existing_project>
+
+Name:					:default
+Created:				2 weeks ago
+Labels:					<none>
+Annotations:				<none>
+Last Modified:				2016-05-03 05:34:49 -0400 EDT
+Policy:					<none>
+RoleBinding[admin]:
+					Role:			admin
+					Users:			fred@example.com <1>
+					Groups:			<none>
+					ServiceAccounts:	<none>
+					Subjects:		<none>
+RoleBinding[dedicated-project-admin]:
+					Role:			dedicated-project-admin
+					Users:			alice@example.com, bob@example.com <2>
+					Groups:			<none>
+					ServiceAccounts:	<none>
+					Subjects:		<none>
+...
+----
+<1> The *fred@example.com* user is a normal, project-scoped administrator of this
+project.
+<2> The *alice@example.com* and *bob@example.com* users are cluster administrators.
+====
+
+[[gs-dedicated-admin-granting-permissions]]
+== Granting Permissions to Other Users
+
+I want to grant permissions to a user or team to be able to do X.
+
+[[gs-dedicated-admin-creating-service-accounts]]
+== Creating Service Accounts
+
+I want to create a service account to be able to run applications like Jenkins
+that make calls back to OpenShift.
+
+[[gs-dedicated-admin-adding-or-removing-default-image-streams-and-templates]]
+== Adding or Removing Default Image Streams and Templates
+
+I want to add/remove image streams and templates to the “openshift” project.
```

Comment:
Should this be *openshift* project here to match the rest of the instances in this PR?